e2e/cypress/integration/integrations/outgoing_webhook/delete_outgoing_webhook_spec.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+let testTeam;
+
+describe('Integrations', () => {
+    before(() => {
+        // # Create test team, channel, and webhook
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+
+            // # Visit the webhook page
+            cy.visit(`/${team.name}/integrations/outgoing_webhooks/add`);
+        });
+    });
+
+    it('MM-T617 Delete outgoing webhook', () => {
+        // # Create an outgoing webhook with trigger word "testing"
```

Comment:
I do not think there is a cypress command for creating an outgoing web-hook so I created manually,

Reply:
@MikeworX You can use `cy.apiCreateWebhook(hook, false);`

Reply:
Thanks!